libs/neo4j/pyproject.toml

```diff
@@ -16,6 +16,7 @@ python = ">=3.9,<4.0"
 langchain-core = "^0.3.0"
 neo4j = "^5.25.0"
 langchain-community = "^0.3.3"
+langchain-openai = "^0.2.8"
```

Comment:
Should this dependency be optional?

Reply:
https://github.com/langchain-ai/langchain-neo4j/pull/10/commits/e3c7e2504e869d815ffa3ac3215d06a241d4756c